src/models/v2/info.ts

```diff
@@ -0,0 +1,32 @@
+import { InfoInterface } from "models/info";
+import { V2BaseModel } from "./base";
+import { V2Contact } from "./contact";
+import { V2License } from "./license";
+
+export class V2Info extends V2BaseModel implements InfoInterface {
+    title(): string {
+        return this.json("title");
+    }
+
+    version(): string {
+        return this.json("version");
+    }
+
+    description(): string {
+        return this.json("description");
+    }
+
+    termsOfService(): string {
+        return this.json("termsOfService");
+    }
+
+    contact(): V2Contact | undefined {
+        const doc = this.json("contact");
+        return doc && new V2Contact(doc);
+    }
+
+    license(): V2License | undefined {
+        const doc = this.json("license");
+        return doc &&  new V2License(doc);
+    }
```

Comment:
`contact()` should return `ContactInterface` not `V2Contact` etc

Reply:
It seems it is not necessary to do that since the returned type already implements the interface. 
I'm coming from Go land, where it is standard to return the implementation types rather than the interface ones (even though from my point of view, I could find pros and cons in all ways).
I couldn't find examples of this in TS besides a few. I'm happy to change it if you consider it should be done.